### PR TITLE
Add card set things

### PR DESCRIPTION
**Pull request by lily-pytel**
Started working on the simplest way to add the set filter to the cards and deck builder pages. Looks like all the images are under core, which is cool - less headache.

### CODE REVIEW

**Comment by alexmiddeleer** [comment]
Less headache for now at least 😸

File: next/lib/card-queries.js [hunk 1]
```diff
@@ -120,7 +127,7 @@ export const executeCardQuery = (
       defenseGTE: defenseGTE || null,
       supertypes:
         supertypes && supertypes.length
-          ? supertypes.map(s => s.toUpperCase())
+          ? supertypes.map((s) => s.toUpperCase())
```
Comment (alexmiddeleer) [inline next/lib/card-queries.js:130]: Don’t need the parens

File: next/lib/card-queries.js [hunk 2]
```diff
@@ -69,14 +72,14 @@ export const getCardsQuery = () => {
...
-export const intEnumsToGQLVars = manaCostEnums => {
+export const intEnumsToGQLVars = (manaCostEnums) => {
   if (!(manaCostEnums && manaCostEnums.length)) {
     return [null, null];
   }
 
   const discreteManaCosts = [];
   let manaGTE;
-  manaCostEnums.map(num => {
+  manaCostEnums.map((num) => {
```
Comment (alexmiddeleer) [inline next/lib/card-queries.js:82]: No parens needed. Your prettier version change?

**Review by alexmiddeleer — APPROVED** [review]
Wow you’re fast!

File: next/lib/card-queries.js (hunk 2 above)
Comment (lily-pytel) [reply next/lib/card-queries.js:82]: New computer - I fixed it but forgot these files so thanks for the catch!

**Comment by lily-pytel** [comment]
I love the vote of confidence but gonna add tests and stuff as well 😆

**Comment by lily-pytel** [comment]
We might need to change the import card script since I just modified it blind but gonna merge anyway

**Pull request merged**